lib/manageiq/rpm_build/build_copr.rb

```diff
@@ -22,11 +22,14 @@ def generate_rpm
         Dir.chdir(RPM_SPEC_DIR) do
           generate_spec_from_template
 
+          rpm_dir = BUILD_DIR.join("rpms")
+          srpm_dir = rpm_dir.join("src")
+          shell_cmd("rpmbuild -bs --define '_sourcedir #{RPM_SPEC_DIR}' --define '_srcrpmdir #{srpm_dir}' #{rpm_spec}")
```

Comment:
should the `shell_cmd` be replaced with equivalent from `AwesomeSpawn` to prevent injections?

Reply:
We could do that in a separate PR since it is used in many places throughout the code base.